cg/apps/gisaid/gisaid.py

```diff
@@ -0,0 +1,126 @@
+"""Interactions with the gisaid cli upload"""
+
+import logging
+from pathlib import Path
+from typing import List
+
+from cg.store import models
+from cg.meta.workflow.fastq import FastqHandler
+from ...utils import Process
+from .models import GisaidSample, UpploadFiles
+from housekeeper.store import models as hk_models
+
+from cg.meta.meta import MetaAPI
+
+LOG = logging.getLogger(__name__)
+
+import csv
+
+
+class GisaidAPI(MetaAPI):
+    """Interface with Gisaid cli uppload"""
+
+    def __init__(self, config: dict):
+        super().__init__(config)
+        self.gisaid_submitter = config["gisaid"]["submitter"]
+        self.gisaid_binary = config["gisaid"]["binary_path"]
+        self.process = Process(binary=self.gisaid_binary)
+        self.fastq_handler = FastqHandler()
+
+    def get_headers(self) -> list:
+        """Get GisaidSample keys as list."""
+
+        return list(GisaidSample.schema().get("properties").keys())
+
+    def get_sample_row(self, sample: models.Sample, family_id: str, fasta_header: str) -> List[str]:
+        """Build row for a sample in the batch upload csv."""
+
+        orig_lab: str = "Stockholm"  # sample.originating_lab
```

Comment:
can we just use those values as defaults in the model? instead of passing

Reply:
No. These values are sample dependent and deed to be stored in the database. We are waiting for the clinichasn to find a solution for how to pass the info via the order. Its not solved yet. Therefor hardcoded now.